pr-data.csv

```diff
@@ -573,11 +573,11 @@ https://github.com/apache/dubbo,5349c13a36d277a090e1dc68fbe7c3b46d78fc90,dubbo-c
 https://github.com/apache/dubbo,5349c13a36d277a090e1dc68fbe7c3b46d78fc90,dubbo-common,org.apache.dubbo.common.utils.PojoUtilsTest.test_total,OD-Vic,Accepted,https://github.com/apache/dubbo/pull/9987,
 https://github.com/apache/dubbo,5349c13a36d277a090e1dc68fbe7c3b46d78fc90,dubbo-common,org.apache.dubbo.common.utils.PojoUtilsTest.test_total_Array,OD-Vic,Accepted,https://github.com/apache/dubbo/pull/9987,
 https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkApplication2,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethod3,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethod4,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethod5,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethods1,UD,,,
-https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethods2,UD,,,
+https://github.com/apache/dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-config/dubbo-config-api,org.apache.dubbo.config.AbstractInterfaceConfigTest.checkInterfaceAndMethod3,UD,Deleted,,https://github.com/apache/dubbo/commit/d0454fdbef0b7724cf5747a276f292eb14c8ab05
```

Comment:
What script did you use to find that the test was Deleted?

Reply:
I did not use a script, I checked the [history](https://github.com/apache/dubbo/commits/3.1/dubbo-common/src/test/java/org/apache/dubbo/config/AbstractInterfaceConfigTest.java) for the class and
I found that the test class was moved to another package [here](https://github.com/apache/dubbo/commit/1be3c999d112215a1354ab396cb2fbb94f3d2a0e) and the tests were deleted [here](https://github.com/apache/dubbo/commit/d0454fdbef0b7724cf5747a276f292eb14c8ab05) so I am updating idoft using the latter commit.

Usually I use these commands to find commits that deleted a file: 
git log --diff-filter=D "*ClassName.java"
Or 'R' for renamed.